ballerina/http_annotation.bal

```diff
@@ -27,6 +27,7 @@
 # + validation - Enables the inbound payload validation functionality which provided by the constraint package. Enabled by default
 # + serviceType - The service object type which defines the service contract. This is auto-generated at compile-time
 # + basePath - Base path to be used with the service implementation. This is only allowed on service contract types
+# + laxDataBinding - Enables or disalbles relaxed data binding on the service side. Disabled by default
```

Comment:
Add more information about the data projection. Check the comment in the client config